src/markdown-pages/instant-observability/build-a-quickstart/create-a-dashboard.mdx

```diff
@@ -0,0 +1,138 @@
+Move to **Dashboards** and click **Create a dashboard** in the top right corner. 
+
+![create a dashboard](../../../images/instant-observability/create-dashboard.png)
+
+Name your dashboard **FlashDB** and click create.
+
+![name your dashboard](../../../images/instant-observability/name-dashboard.png)
+
+## Add charts to your dashboard
+
+Once the dashboard is in place, you can start creating charts. You're going to add the folloiwng charts:
+- Database methods
+- Avergae response time
+- Errors
+- Database size
+- Cache hits
+- Keys
+
+Hover over the dashboard and click **Add a new chart**.
+
+![add a new chart](../../../images/instant-observability/add-new-chart.png)
+
+New Relic One allow you to add charts using query builder or you can choose to add text, images or links using Markdown. Click **Add a chart**.
+
+![add a chart](../../../images/instant-observability/add-a-chart.png)
+
+It will take you to a query builder option. Edit the query as follow:
+
+>```SQL
+>SELECT count(*) FROM fdb_method FACET method 
+>```
+
+![database methods chart](../../../images/instant-observability/fdb-methods.png)
+
+Here, you group your results of FlashDB methods by attribute values and see the count of each type of transaction. You can choose to present your results in different formats such as table, billboard, pie chart etc. 
+
+![chart formats](../../../images/instant-observability/chart-formats.png)
+
+Change the chart type to pie, name your chart and click save.
+
+![pie chart](../../../images/instant-observability/piechart.png)
+
+The chart is now visible on your dashboard.
+
+![piechart dashboard](../../../images/instant-observability/piechart-dashboard.png)
+
+
+You can add more charts to your dashboard following the same pattern. To do so, click **+** in upper right hand corner. 
+
+![add more charts](../../../images/instant-observability/add-more-charts.png)
+
+This will take you to the same _Add to your dashboard_ page, allowing you to add another chart, text, image or links. Add a few more charts to your dashboard as follow:
+
+### Average response time
+Run the following query to observe the **average response time** of database transactions. 
+
+>```SQL
+> SELECT average(fdb_create_responses), average(fdb_read_responses), average(fdb_update_responses), average(fdb_delete_responses) FROM Metric SINCE 30 MINUTES AGO TIMESERIES
+>```
+
+![average response time chart](../../../images/instant-observability/average-response-time.png )
+
+>**Note:** Make sure to click **Run** to see above results. 
+
+Here, you observe average response time for different transactions for past 30 minutes. Click save to add this chart to your dashboard as well. Follow the same procedure to add the following charts as well.
+
+### Errors
+
+>```SQL
+> SELECT sum(fdb_create_errors), sum(fdb_read_errors), sum(fdb_update_errors), sum(fdb_delete_errors) FROM Metric SINCE 60 MINUTES AGO TIMESERIES
```

Comment:
Like the last query, is there value in specifying the `SINCE` clause here over letting the user control it in the UI? It seems like the user can actually change this anyway.